### DISCUSSION TITLE
Question: Is it possible to simultaneously send to multiple influx instances

### DISCUSSION THREAD

**Discussion by anthonysomerset** [i-ky/effluence#12] [Q&A] [answered]
So i understand that its possible to send different data source types to different influx servers/databases etc

is it possible to send a single data type to 2 (or more) influx server/database combinations simultaneously so that they will both recieve the same data?

**Comment by i-ky** [comment, marked as answer]
@anthonysomerset, correct. Module does not support that. But you can use [Telegraf](https://www.influxdata.com/time-series-platform/telegraf/) as a multiplexer. It has [InfluxDB Listener](https://docs.influxdata.com/telegraf/latest/plugins/plugin-list/#influxdb_listener) input plugin which acts like InfluxDB `/write` endpoint and you can configure multiple [InfluxDB v1.x/v2](https://docs.influxdata.com/telegraf/latest/plugins/plugin-list/#influxdb_v2) output plugins as shown [here](https://community.influxdata.com/t/get-telegraf-to-send-to-multiple-influxdb-hosts/3957/2).
